File: 0412-fizz-buzz/solution.cs
public class Solution {
    public IList<string> FizzBuzz(int n) {
        List<string> list = new();

        for(int i=1; i<=n; i++){
            if(i % 3 == 0 && i % 5 == 0){
                list.Add("FizzBuzz");
                continue;
            }
            if(i % 3 == 0) list.Add("Fizz");
            if(i % 5 == 0) list.Add("Buzz");
            
            if(i % 3 != 0 && i % 5 != 0) list.Add(i.ToString());
        }

        return list;
    }
}

File: 0412-fizz-buzz/solution.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> ans = new ArrayList<>();
        String s;

        for(int i=1; i<=n; i++){      
            s = (i%3 == 0 && i%5 == 0)? "FizzBuzz" : (i%5 == 0) ? "Buzz" : (i%3 == 0) ? "Fizz" : String.valueOf(i);
            ans.add(s);
        } 
        return ans;
    }
}
